app/src/main/java/org/simple/clinic/sync/DataSync.kt

```diff
@@ -0,0 +1,76 @@
+package org.simple.clinic.sync
+
+import io.reactivex.Completable
+import org.simple.clinic.ClinicApp
+import org.simple.clinic.bp.sync.BloodPressureSync
+import org.simple.clinic.crash.CrashReporter
+import org.simple.clinic.drugs.sync.PrescriptionSync
+import org.simple.clinic.facility.FacilitySync
+import org.simple.clinic.medicalhistory.sync.MedicalHistorySync
+import org.simple.clinic.overdue.AppointmentSync
+import org.simple.clinic.overdue.communication.CommunicationSync
+import org.simple.clinic.patient.sync.PatientSync
+import org.simple.clinic.protocol.sync.ProtocolSync
+import org.simple.clinic.user.UserSession
+import org.simple.clinic.util.ErrorResolver
+import org.simple.clinic.util.ResolvedError
+import org.simple.clinic.util.exhaustive
+import timber.log.Timber
+import javax.inject.Inject
+
+class DataSync @Inject constructor(
```

Comment:
Should this be `@AppScope` scoped?

Reply:
It's not maintaining any mutable state, so I don't see a reason for retaining this in memory throughout the app's life.

Reply:
Hrm, I don't think so. It's not maintaining any mutable state, so I don't see a reason for retaining this in memory throughout the app's life.

Reply:
👍